docs/Manual/Deployment/Kubernetes/DeploymentResource.md

```diff
@@ -354,6 +354,50 @@ There are two magic values for the secret name:
 - `None` specifies no action. This disables root password randomization. This is the default value. (Thus the root password is empty - not recommended)
 - `Auto` specifies automatic name generation, which is `<deploymentname>-root-password`. 
 
+### `spec.metrics.enabled: bool`
+
+If this is set to `true`, the operator runs a sidecar container for
+every DBserver pod and every coordinator pod. The sidecar container runs
+the ArangoDB-exporter and exposes metrics of the corresponding `arangod`
+instance in Prometheus format on port 9101 under path `/metrics`. You
+also have to specify a string for `spec.metrics.image`, which is the
```

Comment:
As far as I know, if you do not specify the image, it uses the image from the deployment. This is because the exporter should be bundled with the arangodb image.

Reply:
Interesting, I did not know this. However, to the best of my knowledge the exporter is not yet bundled with the release. So an image is needed. I will amend the documentation.

Reply:
Done.